src/lib/pages/account-details/components/asset/index.tsx

```diff
@@ -133,31 +132,34 @@ const AssetSectionContent = ({
   );
 };
 
-export const AssetsSection = ({
-  walletAddress,
-  onViewMore,
-}: AssetsSectionProps) => {
+export const AssetsSection = ({ address, onViewMore }: AssetsSectionProps) => {
   const isMobile = useMobile();
-  const { supportedAssets, unsupportedAssets, isLoading, error } =
-    useUserAssetInfos(walletAddress);
+  const {
+    supportedAssets,
+    totalSupportedAssetsValue,
+    unsupportedAssets,
+    isLoading,
+    totalData = 0,
+    error,
+  } = useUserAssetInfos(address);
   const isMobileOverview = isMobile && !!onViewMore;
 
   if (isLoading) return <Loading withBorder />;
 
-  const totalValue = supportedAssets
-    ? calTotalValue(supportedAssets)
-    : (big(0) as USD<Big>);
   const TotalAssetValueInfo = (
     <UserAssetInfoCard
-      value={totalValue && supportedAssets ? formatPrice(totalValue) : "N/A"}
-      isZeroValue={totalValue.eq(0) || !supportedAssets}
+      value={
+        totalSupportedAssetsValue
+          ? formatPrice(totalSupportedAssetsValue)
+          : "N/A"
+      }
+      isZeroValue={
+        !totalSupportedAssetsValue || totalSupportedAssetsValue.eq(0)
+      }
       helperText="Total Asset Value"
     />
   );
```

Comment:
We can pass `totalSupportedAssetsValue` only and handle N/A case inside component.